package.json

```diff
@@ -115,8 +115,6 @@
     "lodash": "^3.0.1",
     "open": "~0.0.4",
     "ovh-protractor-jasmine2-logs-reporter": "~0.0.1",
-    "ovh-ui-kit": "^2.4.x",
```

Comment:
As bower is deprecated, why don't we keep `ovh-ui-kit` and `ovh-ui-kit-bs` in package.json ?

Reply:
The purpose of this modifications was to unify all the packages either on bower or npm (to avoid duplicate of `ovh-ui-kit`).

Since `ovh-ui-kit` and `ovh-ui-angular` was already on bower, I moved `ovh-ui-kit-bs` there.